crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -149,6 +178,22 @@ impl ContainerNode<u16> {
             handle: DomHandle::new_unset(),
         }
     }
+
+    pub fn is_empty_list_item(&self) -> bool {
+        match self.kind {
+            ContainerNodeKind::ListItem() => {
+                let html = self.to_html();
```

Comment:
I'm a bit uncomfortable with converting to html here - I think `to_html` might have side effects in the future.

But this can be a TODO to fix later if you like.

Reply:
Yeah actually what we need here is a helper that actually just writes out the plain text from all children, this shouldn't be too hard to achieve I suppose. 
Because if I have something like "<li><strong>\u{200b}</strong></li>" I would expect to clear the `<li>` as well.

Reply:
Good point. That is another test case we can create as soon as we have multi-node ranges.